components/Callout/style.scss

```diff
@@ -102,7 +102,7 @@
 }
 @mixin calloutCustomIcons($R: callout) {
   --emoji: 1em;
-  --icon-font: FontAwesome;
+  --icon-font: var(--fa-style-family, 'Font Awesome 6 Pro');
```

Comment:
Wait, will this `--fa-style-family` var allow users to customize the FontAwesome style automagically? And if so, why do we need to add the extra class elsewhere?

Reply:
Yes! This is one way you can customize the icon style, but it would apply across all uses of FA. Would make it harder to mix-and-match. It’s also probably lesser known, since they don’t make it very prominent.

Reply:
Oh sick, so much cleaner than having to add a million extra classes.